### PR TITLE
Minor updates to address lint warnings

### PR DESCRIPTION
**Pull request by mjhenkes**
A pair of minor updates to address lint warnings blocking some PRS from merging. 

Screen Shot 2022-01-19 at 11 38 09 AM

### CODE REVIEW

File: components/PluginsList.vue [hunk 1]
```diff
@@ -24,11 +24,13 @@ export default {
           >{{ pluginType.name }}</a
         >
       </h2>
+      <!-- eslint-disable vue/no-v-html -->
       <p
         v-if="pluginType.description"
         class="my-8"
         v-html="pluginType.description"
```
Comment (mjhenkes) [inline components/PluginsList.vue:31]: The assumption is that this is intentional.

File: components/global/DocsImage.vue [hunk 2]
```diff
@@ -11,6 +11,7 @@ export default {
     },
     title: {
       type: String,
+      default: '',
```
Comment (mjhenkes) [inline components/global/DocsImage.vue:14]: same default as 'alt'

File: components/PluginsList.vue (hunk 1 above)
Comment (cowboy) [reply components/PluginsList.vue:31]: It's not clear to me if this is intentional. Let me check!

**Review by emilyrohrbough — APPROVED** [review]

File: components/PluginsList.vue (hunk 1 above)
Comment (cowboy) [reply components/PluginsList.vue:31]: It is!

**Review by cowboy — APPROVED** [review]

**Pull request merged**